README.md

```diff
@@ -3,15 +3,54 @@
 ## Description:
 --------------
 SAEMI (Size Analysis for Electron Microscopy Images) is a tool for obtaining the size distribution of nanoparticles in electron microscopy images. 
-It uses an image segmentation model trained through a neural network to detect nanoparticle sizes and locations. The neural network used is a u-net
-with a resnet downsample created through the fastai library. From there, it then obtains a histogram of the sizes of each particle, providing 
-important information such as mean particle size, median particle size etc. This tool can be used for characterizing new materials, analyzing 
-environmental effects on chemical systems, ensure quality control for routine synthesis and much more. 
+It uses an image segmentation model trained through a neural network to detect nanoparticle sizes and locations. The neural network used is a 
+<a href="https://arxiv.org/pdf/1505.04597.pdf">u-net</a> with a resnet downsample created through the <a href="https://github.com/fastai/fastai">
```

Comment:
You should learn how to use markdown more thoroughly. You don't need to use HTML tags to create links like this. Something as simple as [u-net](https://arxiv.org/pdf/1505.04597.pdf) does the same thing.